File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String fix = phone_number.substring(0, phone_number.length()-4)
            .replaceAll("[0-9]","*");
		String save = phone_number.substring(phone_number.length()-4);
        
        return fix + save;
    }
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.kt
class Solution {
    fun solution(phone_number: String): String {
        val nLength = phone_number.length
        val sb : StringBuilder = StringBuilder()
        (0 until nLength.minus(4)).forEach { sb.append("*") }
        return sb.append(phone_number.substring(nLength.minus(4), nLength)).toString()
    }
}